### PR TITLE
Refactored handler

### LINKED ISSUE #241: refactor handler

### PR DESCRIPTION
**Pull request by marioradu05**

### CODE REVIEW

File: src/App/assets/js/components/_admin.js [hunk 1]
```diff
@@ -84,7 +84,7 @@ $(document).ready(() => {
     $(document).on('click', '#modalDeleteBtn', () => {
         const selections = $("#bsTable").bootstrapTable('getSelections');
         const messages = adminDeleteModal.find('.modal-messages');
-        request('DELETE', `/admin/delete/${selections[0].uuid}`)
+        request('GET', `/admin/delete/${selections[0].uuid}`)
```
Comment (alexmerlin) [inline src/App/assets/js/components/_admin.js]: Why would we change this from `DELETE` to `GET`?
We should **never** use GET requests to delete a resource.

File: src/App/assets/js/components/_admin.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ $(document).ready(() => {
     $("#adminDeleteBtn").prop('disabled', true);
 
     $(document).on('click', '#adminAddBtn', () => {
-        adminModal.find('.modal-messages').html('');
+        adminModal.find('.modal-messages').html('');``
```
Comment (alexmerlin) [inline src/App/assets/js/components/_admin.js]:
```suggestion
        adminModal.find('.modal-messages').html('');
```

File: src/Admin/src/Controller/AdminController.php [hunk 3]
```diff
@@ -398,7 +379,7 @@ public function listLoginsAction(): ResponseInterface
         return new JsonResponse($result);
     }
 
-    public function logErrors(Throwable|Exception $e, string $message): void
+    private function logErrors(Throwable $e, string $message): void
```
Comment (alexmerlin) [inline src/Admin/src/Controller/AdminController.php:400]: Ok, for now.
In the future, we should make this accessible across all controllers/services - probably as a trait.

File: src/Admin/src/RoutesDelegator.php [hunk 4]
```diff
@@ -19,11 +19,7 @@ public function __invoke(ContainerInterface $container, string $serviceName, cal
         $app->route(
             '/admin[/{action}[/{uuid}]]',
             AdminController::class,
-            [
-                RequestMethodInterface::METHOD_GET,
-                RequestMethodInterface::METHOD_POST,
-                RequestMethodInterface::METHOD_DELETE,
-            ],
+            [RequestMethodInterface::METHOD_GET, RequestMethodInterface::METHOD_POST],
```
Comment (alexmerlin) [inline src/Admin/src/RoutesDelegator.php:22]: See my question on `src/App/assets/js/components/_admin.js` regarding this.

**Review by alexmerlin — CHANGES_REQUESTED** [review]

File: src/App/assets/js/components/_admin.js (hunk 1 above)
Comment (arhimede) [reply src/App/assets/js/components/_admin.js]: @alexmerlin 
This is not an API.
Is a website, which only have GET and POST methods .
Resources we have in API"s , not in websites.

Comment (arhimede) [reply src/App/assets/js/components/_admin.js]: so @MarioRadu refactor de Delete form with a POST form and an extra checkbox for confirmation

**Review by alexmerlin — APPROVED** [review]

**Review by arhimede — APPROVED** [review]

**Pull request merged**